### PR TITLE
Progress - migration to export const

### PR DESCRIPTION
**Pull request by Dishebh**
For issue: #761 

All components under `ignitus-Atoms`, `ignitus-Molecules`, `ignitus-Layout`, `ignitus-Organisms` do not contain export default now.

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@Marvin9 can you please have a look. :)

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index.test.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { mount, render } from 'enzyme';
 
 import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
-import DefaultButtonWithIcon from './defaultButtonWithIcon';
+import { DefaultButtonWithIcon } from '.';
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index.test.tsx:5]: Let's be consistent.

```suggestion
import { DefaultButtonWithIcon } from './index';
```

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.test.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { mount, render } from 'enzyme';
 
 import { AppIcon } from '../../../../..';
-import DefaultIconInput from './defaultIconInput';
+import { DefaultIconInput } from '.';
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.test.tsx:5]: Here too.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Avatar/Components/index.test.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import { Avatar } from './Avatar';
+import { Avatar } from '.';
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Avatar/Components/index.test.tsx:3]: Here as well

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index.test.tsx (hunk 1 above)
Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultButtonWithIcon/Components/index.test.tsx]: Made changes

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.test.tsx (hunk 2 above)
Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultIconInput/Components/defaultIconInput/index.test.tsx]: replaced with index

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Avatar/Components/index.test.tsx (hunk 3 above)
Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Avatar/Components/index.test.tsx]: Done

**Comment by Dishebh** [comment]
@divyanshu-rawat please review :)

**Pull request merged**